alws/crud/platform_flavors.py

```diff
@@ -1,13 +1,40 @@
 from typing import List
 
 import sqlalchemy
+from funcy import pluck
+from sqlalchemy import delete
 from sqlalchemy.future import select
 from sqlalchemy.orm import selectinload
 
 from alws import models
 from alws.schemas.platform_flavors_schema import CreateFlavour, UpdateFlavour
 
 
+async def delete_flavour(db, pf_id: int) -> models.PlatformFlavour:
+    r = await db.execute(
+        'SELECT repository_id FROM platform_flavour_repository WHERE flavour_id=:flavour_id',
+        {'flavour_id': pf_id},
+    )
+    repos_to_delete = set(pluck("repository_id", r.all()))
+
+    await db.execute(
+        'DELETE FROM platform_flavour_repository WHERE flavour_id=:flavour_id',
+        {"flavour_id": pf_id},
+    )
```

Comment:
Maybe we should use 
`select(models.FlavourRepo).where(...) ` and `delete(models.Flawour).where(...) `just to be consistent

Reply:
I'm not familiar with sqlalchemy. What equivalent code using `select(models.FlavourRepo.where(...)` to the following current code would be? FlavourRepo looks different from PlatformFlavour. I remember that is the reason I tried raw SQL at first.

```python
db.execute(
        'SELECT repository_id FROM platform_flavour_repository WHERE flavour_id=:flavour_id',
        {'flavour_id': pf_id},
    )
```

Reply:
that should do the trick
```python3
repos_to_delete = (
    await db.execute(
        select(models.FlavourRepo.c.repository_id)
        .where(models.FlavourRepo.c.flavour_id == pf_id)
    )
).scalars().all()
```

Reply:
Thanks, I'll try.